package.json

```diff
@@ -42,9 +42,13 @@
     "keycloak-js": "9.0.0",
     "lodash.clonedeep": "4.5.0",
     "lodash.omit": "4.5.0",
+    "monaco-editor": "0.34.0",
+    "monaco-editor-webpack-plugin": "7.0.1",
+    "monaco-yaml": "4.0.2",
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-error-boundary": "3.1.4",
+    "react-monaco-editor": "0.41.2",
```

Comment:
A recap:
- `react-monaco-editor` is the react wrapper of `monaco-editor`
- `monaco-yaml` is the library to add yaml syntax support to `monaco-editor`
-  `monaco-editor-webpack-plugin` is the necessary plugin to bundle the editor
- `monaco-editor` the editor itself, peer dependency of `react-monaco-editor`